internal/helm/helmadapter/releaser_service.go

```diff
@@ -124,6 +129,33 @@ func (r releaser) Install(_ context.Context, helmEnv helm.HelmEnv, kubeConfig he
 		}
 	}
 
+	clientSet, err := k8sclient.NewClientFromKubeConfigWithTimeout(kubeConfig, time.Second*10)
```

Comment:
I'd add a new collaborator for performing namespace related part (ensure namespace)

Reply:
I can do it, but I don't see much added value in it right now. This is already the lowest level layer in the stack and the helm operations are also working with pure kubeconfig here. Unit tests would benefit from a dependency that can be mocked out, but I think we are better off with integration tests and from that point of view it doesn't matter whether it's a dependency or inline.
I can however move this to a separate function call for more clarity if that is a good compromise.